src/ansys/dpf/core/server_types.py

```diff
@@ -782,6 +784,7 @@ def _create_shutdown_funcs(self):
 
     def shutdown(self):
         if self.live:
+            info = self.info # initializing the info variable (giving access to ip and port): this can be required if start_local_server is called afterwards
```

Comment:
```suggestion
            _ = self.info # initializing the info variable (giving access to ip and port): this can be required if start_local_server is called afterwards
```